modules/mod_ginger_base/templates/head/favicon.tpl

```diff
@@ -1,5 +1,5 @@
-<link rel="apple-touch-icon-precomposed" sizes="144x144" href="/lib/images/apple-touch-icon-144x144.png" />
-<link rel="apple-touch-icon-precomposed" sizes="152x152" href="/lib/images/apple-touch-icon-152x152.png" />
+<link rel="apple-touch-icon" sizes="144x144" href="/lib/images/apple-touch-icon-144x144.png" />
+<link rel="apple-touch-icon" sizes="152x152" href="/lib/images/apple-touch-icon-152x152.png" />
```

Comment:
Nope not at all, precomposed means that it won't get the styles apple puts around it automatically. So no problems here.